### ISSUE TITLE
Mixture of numeric locales in figure if language of firefox is set to something different than English

### ISSUE THREAD

**Issue by jmozmoz** [mljar/supertree#28]
With Edge or Firefox set to a German user interface, the following code produces the shown figure:

```python

from sklearn.tree import DecisionTreeClassifier
from sklearn.datasets import load_iris
from supertree import SuperTree # <- import supertree :)
import numpy as np

# Load the iris dataset
iris = load_iris()
X, y = np.repeat(iris.data, 100, axis=0), np.repeat(iris.target, 100, axis=0)

# Train model
model = DecisionTreeClassifier()
model.fit(X, y)

# Initialize supertree
super_tree = SuperTree(model, X, y, iris.feature_names, iris.target_names)

# show tree in your notebook
super_tree.show_tree()
```

![grafik](https://github.com/user-attachments/assets/bcc55e07-ef72-4f60-96b4-cb31d53a2f15)

In the red ellipse, a comma is printed to separate 1000: `2.600`. So the number shown is 2600. In the blue ellipse, a point is shown to separate 1000 as in German: `5.000`. The number shown is 5000.

It does not help to change the locale settings within Python, e.g. by adding

``` python
import locale
locale.setlocale(locale.LC_ALL, ("en_US", 'UTF-8'))
```

Changing the user interface language of Firefox to English "solves" the problem. 

So in any case, the used locales for displaying the numbers are inconsistent. Is there a way, to specify them explicit?

It looks like, the setting is done in the Javascript side of the code. 

By the way, the example also demonstrates, that the boxes of the graphs seem to be hard coded and do not consider the axis "ticks" (matplotlib speak). Is there a way to e.g. enforce using "scientific" formating of the numbers of the axis? Or change the size of the boxes dynamically?

**Comment by pplonski** [comment]
Thank you @jmozmoz for reporting the issue. It looks lie there is locale bug.

Regarding, axis ticks formatting. Let's create a new issue to track this.

By the way, are you happy with trees from supertree? How have you found the package?

**Comment by jmozmoz** [comment]
> Thank you @jmozmoz for reporting the issue. It looks lie there is locale bug.
> 
> Regarding, axis ticks formatting. Let's create a new issue to track this.
> 
> By the way, are you happy with trees from supertree? How have you found the package?

Actually colleagues found supertree just by googling. The main advantage is that one can produce "nicer" tree views for presentation which still look similar to the results of  sklearn. The original idea was to plot trees horizontally, meaning the root to the left and the the tree/branches/leaves towards right. We didn't manage this, but the results are still good for presentations.

**Comment by Marchlak** [comment]
Thank you for pointing out this @jmozmoz  I forgot to add formatting to the text in the pie charts. Now, the entire Supertree should have the following structure: commas for thousands (and none in tooltips) and dots for decimal fractions regardless of locale @pplonski  should release a new version of Supertree shortly on pypi. If you encounter any further issues, feel free to reach out.

**Comment by pplonski** [comment]
Thank you @Marchlak, good job. New version `0.5.4` released on PyPi.
